connectivity/drivers/cellular/TELIT/ME310/TELIT_ME310_CellularStack.cpp

```diff
@@ -24,6 +24,12 @@
 static const int sslctxID = 1;
 
 using namespace mbed;
+using namespace std::chrono_literals;
+
+
+namespace mbed {
+constexpr auto socket_timeout = 1s;
+}
```

Comment:
```suggestion
constexpr auto socket_timeout = 1s;
```
I suggest the above to remove the need to specify the namespace at point of use, unless there is a reason to put it in the `mbed` namespace.

Reply:
Can you instead add it as `private static constexpr` variable to the class (in the header file)?
This way it is clear to the class user what the timeout is just by looking at the header file.

Reply:
for that I would have to import std::chronological_literals in the header to use operator""s the same way
importing a complete namespace especially from std is not recommended in a header since everyone using the header would import that namespace too. if you insist I can do it.

Reply:
I do not insist 😌.
Thanks.